novuhq/website
# Thank you page cleanup

This pull request brings design fixes for thank-you page.

**Steps to test:**

- open [preview](https://deploy-preview-252--novu-website.netlify.app/thank-you-discovery-session/)
- check that everything works as expected
